corehq/apps/reports/generic.py

```diff
@@ -983,25 +949,44 @@ def export_table(self):
         3. str(cell)
         """
         headers = self.headers
-        def _unformat_row(row):
-            def _unformat_val(val):
-                if isinstance(val, dict):
-                    return val.get('raw', val.get('sort_key', val))
-                return self._strip_tags(val)
-
-            return [_unformat_val(val) for val in row]
-
         table = headers.as_export_table
         self.exporting_as_excel = True
-        rows = (_unformat_row(row) for row in self.export_rows)
+        rows = (self._unformat_row(self, row) for row in self.export_rows)
         table = chain(table, rows)
         if self.total_row:
-            table = chain(table, [_unformat_row(self.total_row)])
+            table = chain(table, [self._unformat_row(self, self.total_row)])
         if self.statistics_rows:
-            table = chain(table, [_unformat_row(row) for row in self.statistics_rows])
+            table = chain(table, [self._unformat_row(self, row) for row in self.statistics_rows])
 
         return [[self.export_sheet_name, table]]
 
+    @property
+    def export_table_parts(self):
+        """
+        JSON compatible version of export_table only used for export_all_rows_task calls.
+        export_table will be established in the task with the parts returned from this method.
+        """
```

Comment:
If the purpose of this method is to return a json compatible exported report, I'd prefer a name like `export_report_as_json`, with the possible refactor of the `export_table` method to be `export_report_as_excel_table`.

Additionally, I don't think you need to mention that this method is only used in the `export_all_rows_task` as that makes this documentation prone to becoming out of date if someone uses this elsewhere. Also a description of what this method returns, similar to what you mentioned in this PR description would be useful.